matlab/CARFAC_IHC_Step.m

```diff
@@ -16,14 +16,18 @@
 % See the License for the specific language governing permissions and
 % limitations under the License.
 
-function [ihc_out, state] = CARFAC_IHC_Step(bm_out, coeffs, state);
+function [ihc_out, state, receptor_potential] = CARFAC_IHC_Step(bm_out, coeffs, state);
```

Comment:
To pur an alternative here: 

Could/should the receptor_potential be a new member of the ihc_state here? which would simplify the returned items, etc.

Reply:
Yes, it could be, but in the v3 situation it's the main IHC output, and input to SYN, so I didn't prefer to have it buried in the state struct.